build_functions/c-pal_suppressions.sup

```diff
@@ -1,9 +1,3 @@
-{
-   <epoll thread suppression>
-   Helgrind:Race
-   fun:thread_worker_func
```

Comment:
>thread_worker_func [](http://example.com/codeflow?start=7&length=18)

Cannot help but noticing that .c, ut and int are changed but no changes in the .md...